### PR TITLE
completed html-block-and-inline-Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**

### CODE REVIEW

File: html-block-and-inline/html-block-and-inline-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- How do block-level elements affect the document flow?
+  I am able to create paragraphs in multiple layers.
+- How do inline elements affect the document flow?
+  Specifies the text to be seperate from the paragraph.
+- What are the default width and height of a block-level element?
+
+- What are the default width and height of an inline element?
+
```
Comment (shawnkost) [inline html-block-and-inline/html-block-and-inline-quiz-notes.md]: Missing quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing quiz questions, and don't forget to include screenshots in your pull request!

**Comment by GCarolyne** [comment]
Screenshot 2024-10-30 at 8 59 33 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**